test/test_other.py

```diff
@@ -92,6 +92,20 @@ def metafunc(self, wasmfs):
   return metafunc
 
 
+def also_with_wasm_longjmp(f):
```

Comment:
Can we move https://github.com/emscripten-core/emscripten/blob/e5e97d2acf9a04731096c25094c908f0298e0724/test/test_core.py#L127-L149 to `common.py` and use this instead?

Reply:
I did try that but it didn't work.   I think it was because this new test is specifically setjmp/longjmp and not about exception handling.  Indeed the test for this, and the libraries themselves are not written in C++ but in C.

I can't remember the exact failure now.. but I think this way is more precise anyway.

Reply:
Oh sorry I was thinking of `with_both_eh_sjlj`.  

Done!